### ISSUE TITLE
Support for HTML and/or Markdown in the display of output:text with possibly an output:type attribute

### ISSUE THREAD

**Issue by drushbrook** [watson-virtual-agents/chat-widget#80]
For a recent demonstration of WVA I had to include images as part of the message response.

I ended up creating a custom layout called show-html and moved the contents of the output.text into layout.data with the html tags.  By doing so the output.text was blank and hence other interfaces other than the widget did not show the content.

The feedback was good however they were puzzled why there was no out of the box support.

I note that you parse for new lines, html and email in
https://github.com/watson-virtual-agents/chat-widget/blob/master/src/utils/writeMessage.js

**Comment by ethanwinters** [comment]
Yes! This is something we have discussed and is in our list of things to do... we were just waiting for someone to ask for it so it became of higher priority.

**Comment by callmekatootie** [comment]
I am not sure if this is the same but I am trying to render links inside the Virtual Agent.

While my response from the Conversation services is `<a href="http://foobar.com">Link</a>`, instead of showing up as an actual hyperlink, the html mark up is shown.

If this ticket speaks of the same issue, then I would be interested in getting this feature in.
